astroplan/core.py

```diff
@@ -1342,14 +1343,14 @@ def __init__(self, low, high):
         raise NotImplementedError
 
 
-class AboveAirmass(Constraint):
+class AirmassRange(Constraint):
     """
-    An object containing an airmass lower limit.
+    Constrain the airmass within a given range.
     """
 
-    def __init__(self, low):
+    def __init__(self, upper_limit, lower_limit=1):
```

Comment:
@eteq I predict a PR in ~8 months requesting ability to specify a minimum airmass :)

I have no opinion on the options here, but if I were to code this up myself, for consistency with other constraints I would probably name it `AirmassRange` and do `AirmassRange(None, 2)` if I wanted an airmass less than 2 (and would name the kwargs `min` and `max` as @cdeil suggested).